tests/kernel/install_ltp.pm

```diff
@@ -224,12 +224,24 @@ sub add_ltp_repo {
     zypper_ar($repo, name => 'ltp_repo');
 }
 
-sub install_from_repo {
-    my $pkg = get_var('LTP_PKG', (want_stable && is_sle) ? 'qa_test_ltp' : 'ltp');
+sub get_default_pkg {
+    return (want_stable && is_sle) ? 'qa_test_ltp' : 'ltp';
+}
 
-    zypper_call("in --recommends $pkg");
-    script_run "rpm -qi $pkg | tee /opt/ltp_version";
-    assert_script_run "find " . get_ltproot() . q(/testcases/bin/openposix/conformance/interfaces/ -name '*.run-test' > ~/openposix-test-list);
+sub install_from_repo {
+    my $default_pkg      = get_default_pkg;
+    my @pkgs             = (get_var('LTP_PKG', $default_pkg));
+    my $install_both_pkg = $pkgs[0] eq 'both';
+    my $want_32bit       = undef;
+
+    @pkgs = ($default_pkg, $default_pkg . '-32bit') if ($install_both_pkg);
+
+    for my $pkg (@pkgs) {
+        $want_32bit = $pkg =~ m/32bit/ if ($install_both_pkg);
```

Comment:
why do you need this extra check, could there be a case where the package contain `32bit` but you wanna run it in 64bit ?
```suggestion
        $want_32bit = $pkg =~ m/32bit/;
```

Reply:
Well, I could create `pkgs` as hash with package name and prefix keys. And when I'm adding 32bit package I could define also path. Then the extra check would not be needed.
But I should probably define loop outside of the function to simplify things.

Reply:
@cfconrad Whole thing is working, package fixed. I wonder if I should rewrite this bit, or just ignore it and merge.